mogp_emulator/tests/test_Kernel.py

```diff
@@ -0,0 +1,1103 @@
+import numpy as np
```

Comment:
Perhaps split some of the longer tests

Reply:
I changed this file a bunch in another fix (PR fix/inputderiv), will do it there.